pkg/analyzer/analyzers/elevenlabs/result_output.json

```diff
@@ -0,0 +1,527 @@
+{
+    "AnalyzerType": 4,
```

Comment:
I think this should be 6?

Reply:
This was my first PR, but I guess others got merged before this one 😄 I'll fix it